contrib/Profiler/README.md

```diff
@@ -0,0 +1,25 @@
+# 1.Overview
+
+The Profiler can collect the usage of the hardware while your other program is running.
+You can run the Profiler when you want to collect and analyze the information
+
+1. Run your algorithm model.
+2. Run the Profiler when you need start collecting.
```

Comment:
it's better to use `lower_snake_case` instead of `CamelCase` for python module name (file name)